### PR TITLE
Annotation path change due to compiler v2 changes

### PR DESCRIPTION
**Pull request by davidhunglam**

### CODE REVIEW

**Review by beckermarc — CHANGES_REQUESTED** [review]
I think https://github.com/SAP-samples/cloud-cap-samples-java/blob/c7f9d4ee7dbbb7116881d98f7ab79b6bfb457ab6/app/admin/fiori-service.cds#L90 should be adapted as well..

**Comment by davidhunglam** [comment]
> I think
> 
> https://github.com/SAP-samples/cloud-cap-samples-java/blob/c7f9d4ee7dbbb7116881d98f7ab79b6bfb457ab6/app/admin/fiori-service.cds#L90
> 
> should be adapted as well..

It's not a UI annotation, and the CDS tools give me an error when trying to change this to `.texts`.

**Comment by beckermarc** [comment]
What exactly is the error you are getting? I thought the `@ValueList` annotation is relevant for the UI isn't it?

**Comment by davidhunglam** [comment]
You're right that it's relevant for the UI, but I just meant it's not a `@UI` annotation which is referred to by a reference facet.

In the IDE I got red markers on the locale, text and description fields, saying that they cannot be found. But I just tested running it anyways, and it seems to work, nonetheless.

However, leaving it as `_text` there was no complain by the compiler either... but okay, I'll change it.

File: app/admin/fiori-service.cds [hunk 1]
```diff
@@ -61,9 +61,9 @@ annotate AdminService.Books with @(UI : {
 //
 
 annotate my.bookshop.Books with @fiori.draft.enabled;
-annotate AdminService.Books with @odata.draft.enabled;
+annotate AdminService.Books with @odata.draft.enabled;
```
Comment (beckermarc) [inline app/admin/fiori-service.cds:64]:
```suggestion
annotate AdminService.Books with @odata.draft.enabled;
```

**Review by beckermarc — APPROVED** [review]

**Pull request merged**